launcher/ui/widgets/InfoFrame.cpp

```diff
@@ -46,7 +46,7 @@
 void setupLinkToolTip(QLabel* label)
 {
     QObject::connect(label, &QLabel::linkHovered, [label](const QString& link) {
-        if (!link.isEmpty() && !link.startsWith("http"))
+        if (auto url = QUrl(link); !url.isValid() || (url.scheme() != "http" && url.scheme() != "https"))
```

Comment:
that's niiiice

Reply:
You created another review lol

Reply:
lol not used to that really mattering 😂

Reply:
it was added as a single comment -\{:)_/-

Reply:
I think I'll just delete it lol